free-programming-books.md

```diff
@@ -948,22 +937,9 @@ Original Source: [List of freely available programming books](http://web.archive
 
 
 ### Force.com
-* [Analytics Workbook](http://www.salesforce.com/us/developer/docs/workbook_analytics/index.htm)
-* [Apex Design Patterns and Best Practices](http://www.gobookee.org/get_book.php?u=aHR0cDovL3d3dy5zdW5kb2dpbnRlcmFjdGl2ZS5jb20vd2hpdGVwYXBlcnMvU1VOX0NodWNrX0FwZXhkZXNpZ25wYXR0ZXJucy5wZGYKQXBleCBEZXNpZ24gUGF0dGVybnMgYW5kIEJlc3QgUHJhY3RpY2VzIC0gU3VuZG9n)
-* [Apex Workbook](http://www.salesforce.com/us/developer/docs/apex_workbook/index.htm)
-* [Cloud Flow Designer Workbook](http://www.salesforce.com/us/developer/docs/workbook_flow/index.htm)
-* [Database.com Workbook](http://www.salesforce.com/us/developer/docs/workbook_database/index.htm)
 * [Force.com Fundamentals](http://wiki.developerforce.com/page/Force_Platform_Fundamentals)
-* [Force.com Integration Workbook](http://www.salesforce.com/us/developer/docs/integration_workbook/index.htm)
 * [Force.com Platform Fundamentals: An Introduction to Custom Application Development in the Cloud](http://www.lulu.com/shop/salesforcecom/forcecom-platform-fundamentals/ebook/product-17381451.html)
-* [Force.com Workbook](http://www.salesforce.com/us/developer/docs/workbook/index.htm)
 * [Heroku Postgres](http://media.developerforce.com/workbooks/HerokuPostgres_Workbooks_Web_Final.pdf) (PDF)
-* [ISVForce Workbook](http://www.salesforce.com/us/developer/docs/workbook_isv/index.htm)
-* [Security Workbook](http://www.salesforce.com/us/developer/docs/workbook_security/index.htm)
-* [Service Cloud Workbook](http://www.salesforce.com/us/developer/docs/workbook_service_cloud/index.htm)
-* [Site.com Workbook](http://www.salesforce.com/us/developer/docs/workbook_siteforce/index.htm)
-* [Visualforce Workbook](http://www.salesforce.com/us/developer/docs/workbook_vf/index.htm)
```

Comment:
All these removed salesforce.com books have moved. Instead of reintroducing all of them with their new locations, I suggest a link to the force.com index page which lists and links them all. Something like `[Developer Workbooks](http://developer.force.com/workbook) (meta-list)`?